### ISSUE TITLE
List of pair_coeff not read

### ISSUE THREAD

**Issue by UnaiAizpu** [ICAMS/calphy#158]
Hello! I encountered an issue when trying to use an hybrid pair_style (overlay in my case but I checked with others and the problem remains). 

After defining my pair_style and pair_coeff as:
  pair_style: "hybrid/overlay coul/long 11.0 eam/alloy"
  pair_coeff: ["* * eam/alloy /path/to/myfile elt1 elt2","* * coul/long "] 
calphy fails to load both potentials I want to overlay and only writes the first. The simulation then stops with an error "Pair hybrid sub-style coul/long is not used (src/pair_hybrid.cpp:604)".

I think I have an idea of where the problem might be but I am not 100% sure.
I think that before the command  
if self.fix_potential_path:
            self.pair_coeff = self.fix_paths(self.pair_coeff)

there should be a _pair_coeff_names.append(pc_split[0]) for pc in self.pair_coeff: in the same fashion that it is done above for the pair_styles. 

To sum up, what I think is missing is a for loop that would loop the strings contained in the pair_coeff command.

I am no expert and new to the package so I might just be wrong.
Anyway thanks for the package, it is a great tool!
![Capturecalphy](https://github.com/user-attachments/assets/5922e5aa-b6d8-4c5b-86da-9a72eac0c90c)

**Comment by srmnitc** [comment]
@UnaiAizpu Thanks for reporting. Unfortunately hybrid overaly cannot be used at the moment, since calphy also uses overlay internally which leads to issues. Fixing this is in the pipeline.

**Comment by srmnitc** [comment]
@UnaiAizpu This issue is now fixed in v 1.7.0. The potential should be specified as:

```
pair_style:
  - "coul/long 11.0"
  - "eam/alloy"
pair_coeff:
  - "* * coul/long"
  - "* * eam/alloy /path/to/myfile elt1 elt2"
pair_mode: overlay
```
